lighthouse-core/test/computed/metrics/speed-index-test.js

```diff
@@ -23,7 +25,46 @@ describe('Metrics: Speed Index', () => {
       timing: Math.round(result.timing),
       optimistic: Math.round(result.optimisticEstimate.timeInMs),
       pessimistic: Math.round(result.pessimisticEstimate.timeInMs),
-    }).toMatchSnapshot();
+    }).toMatchInlineSnapshot(`
+      Object {
```

Comment:
we initially committed this snapshot before jest supported inline snapshots, which are greatly preferred for objects of this size to make the expectations more obvious and easier to notice

Reply:
aside: did jest mess with the format of the rest of the file? If so, could you try updating to the latest jest and see if they've fixed it? I know it was on their radar.

Reply:
prettier's API doesn't support it yet https://github.com/prettier/prettier/issues/5807

it only affected the 1 line that would be updated on any future `-u` so didn't seem like a big deal 🤷‍♂️

Reply:
darn, still not done.

it's a much larger annoyance for other files, requiring you to either change dozens of other lines to prettier's standards, or copy and pasting the result `-u`, revert everything, and individually applying the new snapshot.

Reply:
It is, and I'd obvs love to adopt prettier even if it's only in our test files, but some `git checkout -p` to clean it up definitely beats manually updating all the snapshot expectations :)

Reply:
nice, didn't know checkout could do patches.

Reply:
> _looks around to see if @brendankenny is hiding in the tall grass_

🌾🌿 ⚔️ 🌿🌾